ui/src/app/views/workspace/component.spec.ts

```diff
@@ -93,14 +102,19 @@ describe('WorkspaceComponent', () => {
         WorkspaceComponent,
         WorkspaceNavBarComponent
       ],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
Please remove this, here and in the other test. This is one of the suggested "fixes" when a unit test does not declare a child component. It is better to add the child components to the declarations section of this test.

Let me know if you have questions or hit issues with this.

Reply:
done